### ISSUE TITLE
Improvements to terms of service

### ISSUE THREAD

**Issue by jimsafley** [omeka-s-modules/Collecting#67]
See this [forum post](https://forum.omeka.org/t/collecting-module-tos/12103).

There is interest in changing the terms of service (TOS) from a plain text to a HTML field. I forget why we decided to make TOS plain text, but it seems like a good idea. One reservation is how it would change existing TOS's that were written using unformatted text and new lines.

We'd do this by adding CKEditor JS to the site admin template:
```js
CKEDITOR.inline(document.getElementById(collecting_tos'));
```
And changing the `Content-Type` in `IndexController::tosAction()`:
```php
$response->getHeaders()->addHeaderLine('Content-Type', 'text/html; charset=utf-8');
```

**Comment by jimsafley** [comment]
Actually, the [specific request](https://forum.omeka.org/t/collecting-module-tos/12103/3) is not to make the TOS a HTML field, but to have an option to change the TOS URL itself. This should be possible by adding a "Terms of service URL" field to site settings, and using that on the collecting form. I prefer this solution over the one above.

**Comment by jimsafley** [comment]
Fixed in c5edc9bf97f353a261c2511900c3c37334691aea
